packages/circuit-ui/components/Modal/ModalService.ts

```diff
@@ -29,9 +29,13 @@ export function getKeyboardFocusableElements(
 }
 
 export function getFirstFocusableElement(
-  dialog: HTMLDialogElement,
+  dialog: HTMLElement,
+  skipFirst?: boolean,
```

Comment:
Let's give this a more descriptive name or add a doc comment to explain when & why to use this option (to skip the close button, I assume?).